packages/vm/src/evm/opcodes/functions.ts

```diff
@@ -873,7 +873,43 @@ export const handlers: Map<number, OpHandler> = new Map([
       runState.eei.log(mem, topicsCount, topicsBuf)
     },
   ],
+  // 0xb3: TLOAD
+  [
+    0xb3,
+    function (runState, common) {
+      if (!common.isActivatedEIP(1153)) {
```

Comment:
This check should not be necessary, this opcode should be unreachable if  the EIP is not activated